cli/bin/grain.js

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/env -S node --experimental-wasi-unstable-preview1 --no-warnings
```

Comment:
These are node flags, not v8 flags (so they can't be set via `v8.setFlagsFromString`). Unfortunately there isn't a way to only suppress experimental feature warnings.

Reply:
This won't work on windows...

Reply:
As evident by the failing tests!